assets/js/common/TimeSeriesLineChart/TimeSeriesLineChart.jsx

```diff
@@ -0,0 +1,209 @@
+import React, { useEffect, useState } from 'react';
+import {
+  Chart as ChartJS,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Tooltip,
+  TimeScale,
+  LogarithmicScale,
+  Legend,
+} from 'chart.js';
+import { Line } from 'react-chartjs-2';
+import ZoomPlugin from 'chartjs-plugin-zoom';
+import classNames from 'classnames';
+import 'chartjs-adapter-date-fns';
+
+const AVAILABLE_COLORS = [
+  {
+    line: '#A5EED4',
+    point: '#58CF9B',
+  },
+  {
+    line: '#FEF08A',
+    point: '#FACC15',
+  },
+  {
+    line: '#FECACA',
+    point: '#F87171',
+  },
+  {
+    line: '#BFDBFE',
+    point: '#60A5FA',
+  },
+  {
+    line: '#E5E7EB',
+    point: '#9CA3AF',
+  },
+  {
+    line: '#004DCF',
+    point: '#1273DE',
+  },
+];
+
+ChartJS.register(
+  LinearScale,
+  PointElement,
+  LineElement,
+  TimeScale,
+  Legend,
+  LogarithmicScale,
+  Tooltip,
+  ZoomPlugin
+);
+
+function TimeSeriesLineChart({
+  title,
```

Comment:
We could also sort here by alphabet :)
`  chartRef,
  chartWrapperClassNames,
  className,
  datasets,
  end,
  onIntervalChange,
  start,
  title,
  yAxisLabelFormatter = (value) => value,
  yAxisMaxValue,
  yAxisScaleType = 'linear',`

Reply:
done